src/fuselage/CCPACSFuselageProfile.cpp

```diff
@@ -295,19 +295,15 @@ void CCPACSFuselageProfile::BuildDiameterPoints(DiameterPointsCache& cache) cons
                 cache.end = point;
             }
         }
-        // project into x-z plane
-        cache.end.SetY(0.);
-        cache.start.SetY(0.);
```

Comment:
@rainman110 fyi, these three lines (298-300) are the only *real* changes to the code.

Reply:
Is it still possible to correctly compute the profile scale for the guide curves?

Reply:
According to the CPACS definition, we should use the "profile height" for scale and I don't know if there is a precise definition for this. 

In TiGL this value is called "diameter" and it is the largest distance between any point on the section to the first point on the section.  The new unit test still checks, wether this value is calculated correctly for one simple CPACS file. 

So in short, the code still works, but there is a some freedom of interpretation of what a "profile height" is, and the TiGL interpretation can yield something that is more like a "profile width", depending on how the profile points are defined.

Reply:
Sounds reasonable. It feels a bit strange, that CPACS does not define it a more precise way, what the scaling actually is. Depending on how this is evaluated, you will end of with very different geometries.

I think we should add a CPACS issue to bring up this issue.

Reply:
See here: https://github.com/DLR-SL/CPACS/issues/666